kinesis.go

```diff
@@ -76,7 +77,7 @@ func makeParams(action string) map[string]string {
 // RequestArgs store params for request
 type RequestArgs struct {
 	params  map[string]interface{}
-	Records []PutRecordsRecord
+	Records []RecordToPut
```

Comment:
Hello everyone. Agree with @harlow - better give name `Record`, it create more sense, than `RecordToPut`.